cmd/kubeadm/test/cmd/completion_test.go

```diff
@@ -40,7 +40,7 @@ func TestCmdCompletion(t *testing.T) {
 				rt.args,
 				actual,
 				rt.expected,
-				(actual == nil),
```

Comment:
What's the motivation of this change?
Is there any document or standard says it's better to remove ths parentheses?

IMO, when the `hack/verify-gofmt.sh` says the format is OK, then the format is OK.

Reply:
When I read the code , there is a  weak warning " Report redundant parentheses expressions" in the Gogland ide. And verify-gofmt.sh can automatically fix the majority of mechanical style issues but not the all issues.In the [https://golang.org/doc/effective_go.html#formatting](url), if the operator precedence hierarchy is shorter and clearer,the parentheses is not necessary.In a word,the format of this Pr is better but not required.

Reply:
Seems good to me, thanks.